src/NServiceBus.PersistenceTests/Sagas/When_persisting_a_saga_with_an_escalated_DTC_transaction.cs

```diff
@@ -52,7 +51,14 @@ public async Task Save_should_fail_when_data_changes_between_concurrent_instance
                         tx.Complete();
                     }
                 }
-            }, Throws.Exception.TypeOf<TransactionAbortedException>());
+            }
+            catch (Exception)
```

Comment:
Seems like the test could still assert that an `Exception` is thrown, since that would actually be true, even if the specific exception type could vary?